distribution/lib/Standard/Database/0.0.0-dev/src/Internal/DB_Wrapped_Error.enso

```diff
@@ -0,0 +1,6 @@
+private
+
+## PRIVATE
+type DB_Wrapped_Error
+    ## PRIVATE
+    Value value
```

Comment:
```suggestion
    Value value

```
We usually try to end our files with a newline.

If you are using IntelliJ, I recommend these settings:
![image](https://github.com/enso-org/enso/assets/1436948/21547222-f112-4941-b30f-82fe270dae5c)

(I often forget about this too 😅)

Reply:
Ack that's the second time I have done that :(. Will find the equivalent in VSCode.

Reply:
Did that work? It looks right locally but not in GitHub...

Reply:
![image](https://github.com/enso-org/enso/assets/1720119/0e14c522-1a5f-434a-bfe5-30f7b6b399d0)

I expected to see a line 7

Reply:
Ah, no the last line is not shown if empty - but it no longer shows the 'No newline at end of file warning'  so it looks good